backend/src/Designer/TypedHttpClients/KubernetesWrapper/KubernetesWrapperClient.cs

```diff
@@ -14,44 +13,31 @@ public class KubernetesWrapperClient : IKubernetesWrapperClient
     private const string PATH_TO_AZURE_ENV = "/kuberneteswrapper/api/v1/deployments";
     private readonly HttpClient _client;
     private readonly ILogger<KubernetesWrapperClient> _logger;
```

Comment:
I know this is not your code, but any reason we "always" pass the logger even though we not use it?

Reply:
It shouldn't be injected on the places where is not used. I removed it now